.github/boring-cyborg.yml

```diff
@@ -144,9 +144,12 @@ labelPRBasedOnFilePath:
         - extensions/jaeger/**/*
     area/jaxb:
         - extensions/jaxb/**/*
+    area/jbang:
+        - **/*JBang*
+        - **/*jbang*
     area/kafka:
-        - extensions/kafka-client/**/*
-        - integration-tests/kafka/**/*
+    - extensions/kafka-client/**/*
+    - integration-tests/kafka/**/*
```

Comment:
This change is weird? Doesn't look consistent with the surrounding items?

Reply:
hmm - gah - stuipd vscode changes indents on paste.fixing.

Reply:
force pushed.